.env.circle-integration-w3t

```diff
@@ -16,12 +16,15 @@ TARGET_NETWORK = 'localhost'
 WEB3TORRENT_URL = 'http://localhost:3000'
 SCREENSHOT_DIR = 'screenshots'
 
+
 ## xstate-wallet
 # This account corresponds to private key 0x187bb12e927c1652377405f81d93ce948a593f7d66cfba383ee761858b05921a
 # That key is provided eth in @statechannels/devtools/src/constants.ts
 HUB_DESTINATION = '0x0000000000000000000000008199de05654e9afa5C081BcE38F140082C9a7733'
 XSTATE_WALLET_DEPLOYER_ACCOUNT_INDEX = '3'
 LOG_DESTINATION = 'pino.log'
+USE_INDEXED_DB = 'true'
```

Comment:
We did consciously omit this previously so we had at least one integration test using the memory backend, but it's probably not that important

Reply:
I think that's probably not the right way to test the memory backend (I'm just thinking about an exponential blowup of test configurations). We could discuss at standup.